File: gradingStudents.js
function gradingStudents(grades){
    const arr = []
    for(let grade of grades) {
        if(grade >= 38) {
            const next_multiple = Number(grade + 5 - grade % 5)
            if(next_multiple - grade < 3) {
                arr.push(next_multiple)
            }
            else {
                arr.push(grade)
            }
        }
        else{
            arr.push(grade)
        }
    }
    return arr
}

File: grading_students.py
def gradingStudents(grades):
    arr = []
    for grade in grades:
        next_multiple = 0
        if grade >= 38:
            next_multiple = int(grade + 5 - grade % 5)
            if next_multiple - grade < 3:
                arr.append(next_multiple)
            else: 
                arr.append(grade)
        else: 
            arr.append(grade)
    return arr
